.github/scripts/create_security_issues.py

```
@@ -0,0 +1,387 @@
+    """
+    try:
+        result = subprocess.run(
+            [
+                "gh",
+                "issue",
+                "list",
+                "--label",
+                "security,automated",
+                "--json",
+                "title",
+            ],
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)